internal/connector/connector.go

```diff
@@ -46,8 +46,6 @@ type Options struct {
 	// This value is sent to the infra API server to update the
 	// Destination.Connection.URL.
 	EndpointAddr types.HostPort
-
-	Kubernetes KubernetesOptions
```

Comment:
The `KubernetesOptions` struct is no longer referenced, you can delete it as well.

Reply:
💥